# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../net-tools/hotfix.patch.musl
# Copyright (C) 2019 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- net-tools-1.60/lib/econet.c.vanilla	2019-02-06 19:47:27.629019566 +0000
+++ net-tools-1.60/lib/econet.c	2019-02-06 19:47:32.601019509 +0000
@@ -23,7 +23,7 @@
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <stdio.h>
-#include <neteconet/ec.h>
+//#include <neteconet/ec.h>
 
 #include "version.h"
 #include "net-support.h"
--- net-tools-1.60/lib/inet_sr.c.vanilla	2019-02-06 19:51:56.245016504 +0000
+++ net-tools-1.60/lib/inet_sr.c	2019-02-06 19:52:38.117016027 +0000
@@ -31,6 +31,7 @@
 #include "intl.h"
 #include "net-features.h"
 #include "util.h"
+#include <asm-generic/param.h>
 
 #if HAVE_NEW_ADDRT
 #define mask_in_addr(x) (((struct sockaddr_in *)&((x).rt_genmask))->sin_addr.s_addr)
--- net-tools-1.60/config.h.vanilla	2019-02-06 19:49:27.249018202 +0000
+++ net-tools-1.60/config.h	2019-02-06 19:53:47.813015232 +0000
@@ -47,13 +47,13 @@
  */
 #define HAVE_AFUNIX 1
 #define HAVE_AFINET 1
-#define HAVE_AFINET6 1
+#define HAVE_AFINET6 0
 #define HAVE_AFIPX 1
 #define HAVE_AFATALK 1
 #define HAVE_AFAX25 1
 #define HAVE_AFNETROM 1
 #define HAVE_AFROSE 1
-#define HAVE_AFECONET 1
+#define HAVE_AFECONET 0
 #define HAVE_AFDECnet 0
 
 /* 
@@ -63,10 +63,10 @@
  */
 #define HAVE_HWETHER 1
 #define HAVE_HWARC 1
-#define HAVE_HWSLIP 1
+#define HAVE_HWSLIP 0
 #define HAVE_HWPPP 1
 #define HAVE_HWTUNNEL 1
-#define HAVE_HWTR 1
+#define HAVE_HWTR 0
 #define HAVE_HWAX25 1
 #define HAVE_HWROSE 1
 #define HAVE_HWNETROM 1
--- net-tools-1.60/netstat.c.vanilla	2019-02-06 19:53:16.953015584 +0000
+++ net-tools-1.60/netstat.c	2019-02-06 19:53:26.949015470 +0000
@@ -85,6 +85,7 @@
 #include <sys/ioctl.h>
 #include <net/if.h>
 #include <dirent.h>
+#include <asm-generic/param.h>
 
 #include "net-support.h"
 #include "pathnames.h"
--- net-tools-1.60/Makefile.vanilla	2019-02-06 19:54:03.729015051 +0000
+++ net-tools-1.60/Makefile	2019-02-06 19:57:27.809012724 +0000
@@ -76,7 +76,7 @@
 NET_LIB_PATH = lib
 NET_LIB_NAME = net-tools
 
-PROGS	:= ifconfig hostname arp netstat route rarp slattach plipconfig nameif ether-wake
+PROGS	:= ifconfig hostname arp netstat route rarp plipconfig nameif ether-wake
 
 -include config.make
 ifeq ($(HAVE_IP_TOOLS),1)
@@ -234,7 +234,6 @@
 	install -m 0755 plipconfig $(BASEDIR)/sbin
 	install -m 0755 rarp       ${BASEDIR}/sbin
 	install -m 0755 route      ${BASEDIR}/sbin
-	install -m 0755 slattach   $(BASEDIR)/sbin
 	install -m 0755 ether-wake $(BASEDIR)/sbin
 ifeq ($(HAVE_IP_TOOLS),1)
 	install -m 0755 ipmaddr    $(BASEDIR)/sbin
